devel/capnproto/files/0007-mutex.c-fix-for-Apple.patch

```diff
@@ -0,0 +1,49 @@
+From a245f0b756d4dfd0aeee4315ee64657ee7a71190 Mon Sep 17 00:00:00 2001
+From: barracuda156 <vital.had@gmail.com>
+Date: Tue, 11 Jul 2023 09:46:00 +0800
+Subject: [PATCH 7/7] mutex.c++: fix for Apple
+
+---
+ c++/src/kj/mutex.c++ | 15 ++++++++++++++-
+ 1 file changed, 14 insertions(+), 1 deletion(-)
+
+diff --git c++/src/kj/mutex.c++ c++/src/kj/mutex.c++
+index 1ddd4921..8f0534f7 100644
+--- src/kj/mutex.c++
++++ src/kj/mutex.c++
+@@ -866,11 +866,17 @@ void Once::reset() {
+   { \
+     int pthreadError = code; \
+     if (pthreadError != 0) { \
+-      KJ_LOG(ERROR, #code, strerror(pthreadError)); \
++      KJ_LOG(WARNING, #code, strerror(pthreadError)); \
```

Comment:
This seems dangerous to me and might lead to weird race conditions in the long run.